tests/regression/juniper/juniper_test.py

```diff
@@ -1009,13 +1009,13 @@ def testProtocolExcept(self):
         self.assertIn('next-header-except tcp;', output, output)
         print(output)
 
-    @capture.stdout
+    # @capture.stdout
```

Comment:
Was this test accidentally disabled?

Reply:
Or rather the capture assertion?

Reply:
It was, those need to be disabled if you want to drop into a debugger.